app/Nova/Attendance.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace App\Nova;
+
+use Laravel\Nova\Panel;
+use Laravel\Nova\Fields\ID;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\HasMany;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\Textarea;
+use Laravel\Nova\Fields\Currency;
+use Laravel\Nova\Fields\MorphTo;
+
+class Attendance extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = 'App\Attendance';
+
+    /**
+     * Get the displayble label of the resource.
+     *
+     * @return string
+     */
+    public static function label()
+    {
+        return 'Attendance';
+    }
+
+    /**
+     * Get the displayble singular label of the resource.
+     *
+     * @return string
+     */
+    public static function singularLabel()
+    {
+        return 'Attendance Record';
+    }
+
+    /**
+     * Get the URI key for the resource.
+     *
+     * @return string
+     */
+    public static function uriKey()
+    {
+        return 'attendance';
+    }
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'name';
+
+    /**
+     * Get the fields displayed by the resource.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function fields(Request $request)
+    {
+        return [
+            new Panel('Basic Information', $this->basicFields()),
+
+            new Panel('Metadata', $this->metaFields()),
+        ];
+    }
+
+    protected function basicFields()
+    {
+        return [
+            Text::make('GTID')
+                ->sortable()
+                ->rules('required', 'max:255'),
+
+            (new BelongsTo('User', 'attendee', 'App\Nova\User'))
+                ->sortable(),
+
+            (new MorphTo('Attended', 'attendable'))
+                ->sortable()
+                ->types([
+                    Event::class,
+                    Team::class,
+                ]),
+
+            (new BelongsTo('Recorded By', 'recorded', 'App\Nova\User'))
```

Comment:
Use `BelongsTo::make` here.

Reply:
I can't use it here and still have the interface refer to it as "Recorded By".

Reply:
Interesting.

Reply:
The make method doesn't pass those through to the constructor as far as I can tell.